SocketIOClientSwift/SocketEventHandler.swift

```diff
@@ -35,6 +35,7 @@ private func emitAckCallbackObjectiveC(socket: SocketIOClient?, num: Int?)
 }
 
 struct SocketEventHandler {
+    let id = NSUUID()
```

Comment:
You can't compare structs. I looked at other Swift-based event dispatchers that support removing specific handlers and they use id's like this to compare them.